README.md

```diff
@@ -1,7 +1,28 @@
-# FortiGate Autoscale Core module
-This is the Autoscale Core module written in TypeScript.
+# Autoscale Core
+The Core Library for the Fortinet Autoscale project.
 
-Note: This is a pre-release for internal development purposes.
+The Autoscale project provides a cloud-based multi-group auto scaling functionality for virtual machines of Fortinet products which have native HA features to form a cluster with failover protection.
+
+This project provides the fundamental, generic logics of a Hybrid Licensing architecture. It also includes implementations for certain popular cloud platforms.
+
+This project has the following features:
+ * Multi-group Hybrid Licensing models. (Hybrid, BYOL-Only, PAYG-Only)
+ * AWS Transit Gateway Integration (see: https://github.com/fortinet/fortigate-autoscale-aws)
+
+
+## Supported platforms
+This project supports autoscaling for the cloud platforms listed below:
+* Amazon AWS
+
+## NodeJS dependency
+
+To use this project as a dependency:
```

Comment:
```suggestion
This project must be used as a NodeJS dependency. At the project root directory, run the following command:
```

Reply:
```suggestion
To use this project as an NPM dependency, install it using the NPM command:
```
I propose another change here, limiting it to the context of "npm dependency".

Because the content is under the header **NodeJS dependency**, probably we don't need to mention NodeJs dependency again here.

Because there are recently two popular ways to install a NodeJS dependency: **NPM** and **YARN**, we only provide the **NPM** way. That's why I includes the **NPM dependency** wording in my second proposal.

The **must be used** in the suggestion sounds like being used as a NodeJS dependency is the only one way to use this project. In fact, people can create their own library that inherits this autoscale-core project and use their project in some different ways.